LEGO1/mxtransitionmanager.cpp

```diff
@@ -165,14 +147,14 @@ MxResult MxTransitionManager::StartTransition(TransitionType p_animationType, Mx
 
     this->m_transitionType = p_animationType;
 
-    m_unk20.bit0 = p_unk;
+    m_copyFlags.bit0 = p_doCopy;
 
-    if (m_unk20.bit0 && m_unk08 != NULL) {
-      m_unk08->vtable54(1);
+    if (m_copyFlags.bit0 && m_waitIndicator != NULL) {
+      m_waitIndicator->Enable(TRUE);
 
-      MxTransitionManagerUnknownSubclass2 *iVar2 = m_unk08->m_unk1c;
-      iVar2->m_unk3c = 10000;
-      iVar2->m_unk30 |= 0x200;
+      MxDSAction *action = m_waitIndicator->GetAction();
+      action->SetLoopCount(10000);
+      action->SetFlags(action->GetFlags() | 0x200);
```

Comment:
Can you add an "unknown" flag to the enum in `MxDSAction` and use that instead of `0x200`? Thank you!